src/AffectedUserFlow/AffectedUserContext.tsx

```diff
@@ -49,9 +49,19 @@ export const AffectedUserProvider: FunctionComponent<AffectedUserProviderProps>
 
   const navigateOutOfStack = () => {
     if (isOnboardingComplete) {
-      navigation.navigate("App", { screen: Stacks.Home })
+      navigation.dispatch(
```

Comment:
@tokumori Does this solve the issue you noticed?

Reply:
@johnschoeman Also want to make you aware of this update. The intention is to reset the navigation state to avoid any of this funky behavior instead of just navigating, which I believe was @tokumori's original intention here.

Reply:
@devinjameson Yes, this solves those issues! Avoiding the funky behavior was exactly my intention with the reset.

Reply:
LGTM! nice work y'all